addons/mail/static/src/models/media_preview/media_preview.js

```diff
@@ -173,6 +196,12 @@ function factory(dependencies) {
         isVideoEnabled: attr({
             compute: '_computeIsVideoEnabled',
         }),
+        videoElementHeight: attr({
+            compute: '_computeVideoElementHeight',
+        }),
+        videoElementWidth: attr({
+            compute: '_computeVideoElementWidth'
+        }),
```

Comment:
We've been a bit lazy recently but ideally we would want all fields to have a docstring.